tests/tgenkey.c

```diff
@@ -67,6 +68,49 @@ static void check_keys(OSSL_STORE_CTX *store)
         exit(EXIT_FAILURE);
     }
 
+    /* check we can get pub params from key */
```

Comment:
Same here, no coverage for the eddsa

Reply:
yeah I think we should have a separate PR for that, this one grew enough

Reply:
Great, thank you